<launch>
 
  <node  pkg="apriltags_ros" type="apriltag_detector_node" name="apriltag_detector">
    <rosparam param="tag_descriptions">[
      {id: 0, size: 0.528000},
      {id: 1, size: 0.160000, frame_id: target11111},
      {id: 25, size: 0.03400, frame_id: target1},
      {id: 35, size: 0.12000,frame_id: marker1}]
    </rosparam>
  </node>

  <node ns="camera" pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen">
    <param name="width" type="int" value="640" />
    <param name="height" type="int" value="480" />
    <param name="format" type="string" value="jpeg" />
    <param name="fps" type="int" value="60" />
    <param name="frame" type="string" value="webcam" />
    <param name="device" type="string" value="/dev/video0" />
    <param name="camera_info_url" type="string" value="file://$(find ffcalib)/ost_c.yaml" />
    <remap from="/camera/camera_info" to="/f_camera/camera_info"/>
  </node>

  <node pkg="image_transport" name="republish_uvc"  type="republish" args="compressed in:=/camera/image_raw _image_transport:=compressed raw out:=/f_camera/image_raw" />
  <node ns="f_camera"  pkg="image_proc" name="image_proc"  type="image_proc" />



 <arg name="COM_Port" default="$(find camera_control)/launch/port_set.yaml"/>
  <rosparam command="load" file="$(arg COM_Port)" />
   <node name="target_path_track" pkg="camera_control" type="target_path_track" output="screen">
    <rosparam command="load" file="$(arg COM_Port)" />
   </node>


</launch>
